CHANGELOG.md

```diff
@@ -30,6 +32,10 @@ Other breaking changes:
 * [CHANGE] Treat custom textfile metric timestamps as errors #769
 * [CHANGE] Drop exec_ in boot_timestamp_seconds on *bsd #839
 * [CHANGE] Use lowercase cpu label name in interrupts #849
+* [CHANGE] Sort collector names in startup logs #857
```

Comment:
This is an enhancement at best

Reply:
I can just drop it from the CHANGELOG if you want.

Reply:
I personally wouldn't mention it, but it's up to you.

Reply:
Done, removed.